src/order/order-request-body.ts

```diff
@@ -39,5 +39,5 @@ export interface OrderPaymentRequestBody {
      * An object that contains the details of a credit card, vaulted payment
      * instrument or nonce instrument.
      */
-    paymentData?: CreditCardInstrument | HostedInstrument | HostedCreditCardInstrument | HostedVaultedInstrument | NonceInstrument | VaultedInstrument | CreditCardInstrument & WithDocumentInstrument | CreditCardInstrument & WithCheckoutcomFawryInstrument | CreditCardInstrument & WithCheckoutcomSEPAInstrument | CreditCardInstrument & WithCheckoutcomiDealInstrument | HostedInstrument & WithMollieIssuerInstrument;
+    paymentData?: CreditCardInstrument | HostedInstrument | HostedCreditCardInstrument | HostedVaultedInstrument | NonceInstrument | VaultedInstrument | CreditCardInstrument & WithDocumentInstrument | CreditCardInstrument & WithCheckoutcomFawryInstrument | CreditCardInstrument & WithCheckoutcomSEPAInstrument | CreditCardInstrument & WithCheckoutcomiDealInstrument | HostedInstrument & WithMollieIssuerInstrument | HostedInstrument & WithAccountCreation;
```

Comment:
Do we need to add `hostedInstrument` here?

Reply:
I don't think actually. But is it will be ok to live `WithAccountCreation` alone here?

Reply:
Yes, I think so it's just a type being mapped and returned from checkout-js I assume?

Reply:
I checked this case. We should use `hostedInstrument` to prevent code duplication in `WithAccountCreation` interface. 

If we remove pairing of `hostedInstrument` and `WithAccountCreation` interfaces we will get a bunch of errors after project build.

Reply:
> Yes, I think so it's just a type being mapped and returned from checkout-js I assume?

Yes, you are right